ext/compressors/zlib/zlib_compress.c

```diff
@@ -32,18 +32,10 @@
 #include <stdlib.h>
 #include <string.h>
 
+#include <wiredtiger_config.h>
 #include <wiredtiger.h>
 #include <wiredtiger_ext.h>
 
-/*
- * We need to include the configuration file to detect whether this extension
```

Comment:
This was called out explicitly, since we don't expect user implemented compressors to do it, and it's not "normal" for applications to include that header file. I don't feel super strongly if you do. I also wonder why the Windows specific define was there if it's not necessary any more.